lib/publiccloud/k8s_provider.pm

```diff
@@ -89,4 +97,21 @@ sub configure_docker {
     $self->provider_client->configure_docker();
 }
 
+=head2 push_container_image
+Upload a container image to the ECR. Required parameter is the
```

Comment:
add an empty line between head and description

Reply:
Also I changed ECR by provider container registry because now is generic